apps/taxes/src/modules/avatax/ui/avatax-configuration-credentials-fragment.tsx

```diff
@@ -121,7 +121,24 @@ export const AvataxConfigurationCredentialsFragment = (
             }
             name="isSandbox"
           />
-
+          <AppToggle
+            control={control}
+            label="Document recording"
+            helperText={
+              <HelperText>
```

Comment:
afaik default helper text has text size "small", please check if we don't have inconsistency